lineman/app/components/thread_page/activity_card/new_comment.haml

```diff
@@ -17,9 +17,7 @@
       %h3.new-comment__in-reply-to{ng-if: 'comment.parentId', id: 'event-{{event.id}}'}
         %strong
           %a{lmo-href-for: 'comment.author()'}> {{ comment.authorName() }}
-        %span{translate: 'new_comment_item.in_reply_to'}
-        %strong
-          %a{lmo-href-for: 'comment.parentAuthor()'}> {{ comment.parentAuthorName }}
+        %span{translate: 'new_comment_item.in_reply_to', translate-value-recipient: '{{comment.parentAuthorName}}'}
```

Comment:
Why are we losing the link here? All that needs to be done to fix this is either add a check to `parentAuthor()` to not error when `parent()` is blank, or change the lmo-href-for to use `comment.parent().author()` instead of `comment.parentAuthor()`